tscached.yaml

```diff
@@ -27,8 +27,9 @@ tscached:
         staleness_threshold: 10  # data up to this far in the past is "new"
 
     chunking:
-        chunk_length: 1800  # chunk on 30 minute intervals
+        chunk_length: 3600  # chunk on 1 hour intervals
         max_chunks: 6  # increase chunk size if more than this needed
+        thread_timeout: 30  # timeout on waiting for chunk threads to join
```

Comment:
We might want to set this to Kairos query timeout + some small duration. 

With chunking of 1 hr, we might still have a very high volume query that returns in > 30 seconds.